### PR TITLE
Adds translations

### PR DESCRIPTION
**Pull request by Kongkille**
Grabbed mostly from #26

### CODE REVIEW

**Comment by Kongkille** [comment]
LGTM

File: web/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="da">
```
Comment (madsnedergaard) [inline web/index.html:2]: Lol, we should fix this!

File: web/src/translation/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  .use(HttpApi)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    fallbackLng: 'en',
+    debug: import.meta.env.DEV,
+    backend: {
+      loadPath: `${LOCALES_PATH}/{{lng}}.json`,
+      crossDomain: true,
+      // request: requestWithXmlHttpRequest, // TODO: test on mobile, if issue check out [INSERT PR]
```
Comment (madsnedergaard) [inline web/src/translation/i18n.ts:52]: You were a bit too fast here ;)

File: web/src/translation/translation.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+export const translate = (key: string, ...arguments_: string[]) =>
+  translateWithTranslator(i18next.t, key, ...arguments_);
+
+// Hook for translations inside React components
+export const useTranslation = () => {
+  const { t, i18n } = useTranslationHook();
+  const __ = (key: string, ...arguments_: string[]) =>
+    translateWithTranslator(t, key, ...arguments_);
+  return { __, i18n };
+};
```
Comment (madsnedergaard) [inline web/src/translation/translation.ts:30]: Maybe we should consider renaming it this time, as it was a bit annoying before that it was called the same as the hook in i18next?

**Pull request merged**